src/Web3/Eth.elm

```diff
@@ -60,14 +258,49 @@ sendTransaction txParams =
         }
 
 
+sendRawTransaction : Bytes -> Task Error TxId
+sendRawTransaction signedData =
+    Web3.toTask
+        { func = "eth.sendRawTransaction"
+        , args = Encode.list [ Encode.string signedData ]
+        , expect = expectString
+        , callType = Async
+        }
+
+
+sign : Address -> Bytes -> Task Error Bytes
+sign address bytes =
+    Web3.toTask
+        { func = "eth.sign"
+        , args = Encode.list [ Encode.string address, Encode.string bytes ]
+        , expect = expectString
+        , callType = Async
+        }
+
+
+call : TxParams -> Task Error TxId
+call =
+    callAtBlock Latest
```

Comment:
Using this pattern a bit to deal with what would normally be optional/default values in Web3.js.

Baking `Latest` BlockId in the`someFunc` version. If people want to customize more, they will call `someFuncAtBlock`.